Makefile

```diff
@@ -448,25 +448,27 @@ staging: doc/CHANGES
 	# Delete the tmp files
 	cd ${TMPDIR} && /bin/rm -rf daikon daikon.dist old_tar.txt new_tar.txt
 
-# Copy the files in the staging area to the website.  This will copy
```

Comment:
This comment is a bit confusing because the staging area is on the website (it's served by the webserver).  I think you mean to the live or main website.